### PR TITLE
Adding Logical layout implementation

### LINKED ISSUE #42: Offer a logical and a physical pin layout
A bunch of the pins (GND, 5V etc) cannot be configured and their levels do not changes, so they really just take up screen space. So, consider a logical pin layout without them that can be chosen.

But it is useful to see the physical pin layout when wiring up things from a header, etc - so a physical layout is also useful.

So, offer a switch in the UI to switch between a physical and logical layout.

### PR DESCRIPTION
**Pull request by sundaram123krishnan**
Fixes #42

### CODE REVIEW

**Review by andrewdavidmackenzie — CHANGES_REQUESTED** [review]
I need to study the UI code more and dive into that area more in the next few days, then maybe will suggest some refactors..... for now, just some minor comments in the review.

File: src/pin_layout.rs [hunk 1]
```diff
@@ -77,20 +106,58 @@ impl Sandbox for Gpio {
...
+            Message::LayoutChanged(layout) => {
+                self.choose_layout = layout;
+            }
         }
     }
 
     fn view(&self) -> Element<Self::Message> {
-        container(pin_view(&self.gpio_description, &self.gpio_config, self))
+        let layout_selector = pick_list(
+            &Layout::ALL[..],
```
[the remark this answers is not in the document]
Comment (sundaram123krishnan) [reply src/pin_layout.rs:117]: yes, the slice type implements Borrow trait

File: src/piggui.rs [hunk 2]
```diff
@@ -15,11 +15,12 @@
     let window = window::Settings {
         resizable: false,
         decorations: true,
+        size: iced::Size::new(800.0, 1000.0),
```
[the remark this answers is not in the document]
Comment (sundaram123krishnan) [reply src/piggui.rs:18]: Default size doesn't fit the logical layout, that's why increased the height

[the remark this answers is not in the document]
Comment (andrewdavidmackenzie) [reply src/piggui.rs:18]: OK. Maybe later we can figure out how to have Iced grow the size to "wrap" the contents, so it's flexible.
But not urgent.

**Review by andrewdavidmackenzie — APPROVED** [review]

**Pull request merged**